lib/game_factory.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../lib/player'
```

Comment:
Can you remove require_relative from all areas of your codebase?

Reply:
Instead please favor altering the load path at the beginning of your application and using require instead.